README.md

```diff
@@ -67,3 +74,25 @@ sudo systemctl enable --now libvirtd
 sudo usermod --append --groups libvirt `whoami`
 vagrant up --no-parallel
 ```
+
+## Developer Workstation Setup (Debian)
+
+```
+sudo apt install virtualenv
+sudo apt install python3-pip libssl-dev
+sudo apt install vagrant-libvirt
+sudo apt install qemu-system libvirt-daemon-system
+
+cd infrastructure_configs
+. venv/bin/activate
```

Comment:
I think you need a `python -mvenv venv` right before this to actually create the virtual environment before activating it.